### PR TITLE
Remove duplicate lin in OpenAI manifest

### PR DESCRIPTION
**Pull request by sebastienros**
The file was auto-generated, but this duplicated line would only disappear if the file was deleted before. It got added but also fixed by running `dotnet run -- --publisher manifest --output-path aspire-manifest.json`
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/aspire/pull/2662)

### CODE REVIEW

**Review by davidfowl — APPROVED** [review]

File: playground/OpenAIEndToEnd/OpenAIEndToEnd.AppHost/aspire-manifest.json [hunk 1]
```diff
@@ -36,12 +36,12 @@
         "http": {
           "scheme": "http",
           "protocol": "tcp",
-          "transport": "http"
+          "transport": "tcp"
```
Comment (davidfowl) [inline playground/OpenAIEndToEnd/OpenAIEndToEnd.AppHost/aspire-manifest.json:39]: tcp?

Comment (sebastienros) [reply playground/OpenAIEndToEnd/OpenAIEndToEnd.AppHost/aspire-manifest.json:39]: Currently debugging, behavior seemed random as I had to delete the file to get this change applied, and it seems it should not be tcp.

**Review by eerhardt — APPROVED** [review]
LGTM - sans the tcp part.

File: playground/OpenAIEndToEnd/OpenAIEndToEnd.AppHost/aspire-manifest.json (hunk 1 above)
Comment (sebastienros) [reply playground/OpenAIEndToEnd/OpenAIEndToEnd.AppHost/aspire-manifest.json:39]: I see the change and the default value. Still can't explain why it didn't get updated when the file existed, or why the other file got a duplicated line until the file was deleted. I switched to a "dev drive", hope this is not a bug.

Comment (davidfowl) [reply playground/OpenAIEndToEnd/OpenAIEndToEnd.AppHost/aspire-manifest.json]: https://github.com/dotnet/aspire/pull/2664

**Pull request merged**